lib/specialist_publisher.rb

```diff
@@ -0,0 +1,7 @@
+module SpecialistPublisher
+  mattr_accessor :publish_pre_production_finders
+
+  def self.should_publish_pre_production_finders?
+    publish_pre_production_finders.present?
+  end
+end
```

Comment:
There's already a `SpecialistPublisher` module defined in `config/application.rb`. Perhaps this could go there? It might be easier to find than in `lib/`? It'd need to go inside the module, but outside the `class Application`.